src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -229,6 +230,15 @@ public void launch(SlaveComputer computer, TaskListener listener) {
         }
     }
 
+    private Watch eventWatch(KubernetesClient client, String podName, String namespace, TaskListener runListener) {
+        try {
+            return client.events().inNamespace(namespace).withField("involvedObject.name", podName).watch(new TaskListenerEventWatcher(podName, runListener));
+        } catch (KubernetesClientException e) {
+            LOGGER.log(Level.FINE, e, () -> "Cannot watch events on " + namespace + "/" +podName);
```

Comment:
Is is reasonable to ask that this log line be written more readily (less restricted than FINE) and include a note/link to instructions for the required level of access needed to watch pod events?

Reply:
Yes, this seems like it could be at `INFO`.

Reply:
Or printed to the `runListener`.